env.py

```diff
@@ -0,0 +1,50 @@
+"""
+Handles all environment variables set by the .env file
+
+Uses pydantic validation to allow for static and runtime type checks. The bot
+will not run unless all required fields are set.
+"""
+
+from pydantic import Field, field_validator
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+
+class _Env(BaseSettings):
+    """
+    THIS IS AN INTERNAL CLASS. DO NOT IMPORT THIS. INSTEAD USE THE `env`
+    INSTANCE FROM THIS SAME MODULE LIKE SO:
```

Comment:
I think prepending the class name with a _ is good enough, no need to rewrite here